tests/testthat/test-module_teal.R

```diff
@@ -1698,8 +1878,7 @@ testthat::describe("srv_teal teal_module(s) transformer", {
     )
   })
 
-  testthat::it("throws warning when transformer return reactive.event", {
-    testthat::expect_warning(
+  testthat::it("does not throw a warning when transformator returns reactive.event", {
```

Comment:
Hey @gogonzo on main we had such a warning for teal_transform_module

https://github.com/insightsengineering/teal/blob/d9fb47dfaef4cfcb0a4086c9105cc0448279b51c/R/teal_data_module.R#L162C1-L171C10

Not it is removed from `teal_transform_module` hence this warning is not shown and the test is irrelevant. Is that intended or an oversight?

Reply:
this might be a result of inaccurate merge

Reply:
Alrighty, would you like me to revert and bring back this warning : )?

Reply:
I will bring it back

Reply:
Reverted back in here 30002e11b7637744f2814a1d23764cfd7378b34a